modules/150-user-authn/openapi/conversions/v2.yaml

```diff
@@ -0,0 +1,3 @@
+version: 2
+conversions:
+  - .publishAPI.enabled = .publishAPI.enable | del(.publishAPI.enable)
```

Comment:
I don't think it works. It will generate
```yaml
spec:
  settings:
    publishAPI:
      enabled: null
```

Reply:
`.publishAPI.enable` has a default value